inst/css/BS4/pkgdown.scss

```diff
@@ -296,42 +297,31 @@ nav[data-toggle="toc"] .nav .nav > li > a {
 }
 
 /* h1 */
-
 h1.display-4 {
-  font-size: 2.5rem;
-}
-
-/* Links color contrast */
-
-a {
-  color: #0054AD;
-}
-
-a:hover {
-  color: #0054AD;
+  font-size: $h1-font-size;
 }
 
-
 /* navbar active link */
 .navbar-light .navbar-nav .active > .nav-link {
```

Comment:
not entirely happy as it should probably depend on the bg as well not only the type.